crates/runtime/src/databackend/postgres.rs

```diff
@@ -141,6 +143,27 @@ impl PostgresBackend {
     }
 }
 
+fn read_pg_config(
```

Comment:
Let's minimize duplication of this logic between the connector and backend as much as possible

Reply:
Move the read_pg_config to datafusion.rs, and it is shared by both data backend and data connector